app/script/view_model/content/MessageListViewModel.js

```diff
@@ -613,7 +613,9 @@ z.viewModel.content.MessageListViewModel = class MessageListViewModel {
   handleClickOnMessage(messageEntity, event) {
     const hasMentions = messageEntity.mentions().length;
     const mentionElement = hasMentions && event.target.closest('.message-mention');
-    if (mentionElement) {
+    const userId = mentionElement.dataset.userId;
+
+    if (mentionElement && userId) {
```

Comment:
A self mention does not contain a user id. The click should not trigger the show user details flow.